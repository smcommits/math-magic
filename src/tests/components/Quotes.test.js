import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../../components/Quote';

describe('App Componenet', () => {
  it('it always renders the same markup', () => {
    const componentTree = renderer
      .create(<Quotes />)
      .toJSON();
    expect(componentTree).toMatchSnapshot();
  });
});
