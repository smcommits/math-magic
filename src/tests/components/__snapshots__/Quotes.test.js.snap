// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Componenet it always renders the same markup 1`] = `
<main>
  <p>
    
        Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. 
        â€“William Paul Thurston
      
  </p>
</main>
`;
